extensions/amp-analytics/0.1/requests.js

```diff
@@ -570,7 +570,7 @@ function reportErrorTemp(url, info) {
     const elementWidth = Number(elementSize[0]);
     const elementHeight = Number(elementSize[1]);
     if (elementWidth == 0 || elementHeight == 0) {
-      dev().error(
+      dev().expectedError(
```

Comment:
Question: How to tell if we can aggregate the logs? By collecting static error messages?